docusaurus/docs/dev-docs/installation/cli.md

```diff
@@ -89,6 +91,34 @@ Experimental Strapi versions are released every Tuesday through Saturday at midn
 Please use these experimental builds at your own risk. It is not recommended to use them in production.
 :::
 
+### Self-hosting your Strapi project
+
+When the installation script runs, the terminal will first ask you if you want to login/signup. Choosing `Login/signup` will create a free, 14-day trial [Strapi Cloud](/cloud/intro#what-is-strapi-cloud) account as described in the [Quick Start Guide](/dev-docs/quick-start).
+
+If you prefer hosting the project by yourself, use the arrow keys to select `Skip`. The script will resume and create a local project. To deploy this project and host it online, you will then have to host the project's code on a repository (e.g., on GitHub) before following a [3rd-party deployment guide](/dev-docs/deployment).
```

Comment:
Users can still use `strapi login` and `strapi deploy` if they skip the cloud part in the create-strapi-app.